src/RTTComms.cpp

```diff
@@ -109,6 +109,12 @@ namespace BrainCloud
 
             delete _socket;
             _socket = NULL;
+            if(_disconnectedWithReason==true)
+            {
+                Json::FastWriter myWriter;
+                std::string output ="\n" + myWriter.write(_msg); 
+                printf(output.c_str());
```

Comment:
I think we wanted to do more than just print it out? I don't remember. Don't want this to be sent to _connectCallback failure?

But at least the user will have log. So leave it like that, we're good. This is a very rare case anyway.